quiche/src/ffi.rs

```diff
@@ -1445,6 +1528,19 @@ pub extern fn quiche_get_varint(
     b.off() as ssize_t
 }
 
+fn optional_std_addr_from_c(
+    addr: *const sockaddr, addr_len: socklen_t,
+) -> Option<SocketAddr> {
+    if !addr.is_null() && addr_len > 0 {
```

Comment:
Slight nit, but I think it would be cleaner as:
```
if addr.is_null() || addr_len == 0 {
    return None;
}

Some({
    let addr = unsafe { slice::from_raw_parts(addr, addr_len as usize) };
    std_addr_from_c(addr.first().unwrap(), addr_len)
})
```

Reply:
> Slight nit, but I think it would be cleaner as:
> 
> ```
> if addr.is_null() || addr_len == 0 {
>     return None;
> }
> 
> Some({
>     let addr = unsafe { slice::from_raw_parts(addr, addr_len as usize) };
>     std_addr_from_c(addr.first().unwrap(), addr_len)
> })
> ```

Let me make it `addr_len <= 0` just to keep things as safe as possible